### ISSUE TITLE
Sorting of solution file?

### ISSUE THREAD

**Issue by magnusottosson** [endjin/git-hooks#12]
We have lots of problems with conflict in the sln file. Any reason for not sorting the solution file?

**Comment by HowardvanRooijen** [comment]
I don't think that was a pain point for us - the churn seemed to be mainly in the csproj files.

Although migrating to the new [Visual Studio Common Project System](https://github.com/dotnet/project-system) has solved the majority of our git related issues.
